alacritty_terminal/src/term/color.rs

```diff
@@ -12,6 +12,9 @@ use crate::ansi::NamedColor;
 /// Number of terminal colors.
 pub const COUNT: usize = 269;
 
+/// Index of the cursor color.
+pub const CURSOR_COLOR_INDEX: usize = 258;
```

Comment:
You shouldn't use a constant. You should use `NamedColor::Cursor as usize`. Otherwise this will explode if we change the order of the colors.

Reply:
Oh, I was trying to look for it and haven't found.